vite/client.mjs

```diff
@@ -66,10 +66,12 @@ export default defineConfig(async (params) => {
       break
 
     case 'production':
-      plugins.push(legacy({
-        ignoreBrowserslistConfig: true,
-        targets: browserslist.production,
-      }))
+      if (viteEnv.DISABLE_LEGACY_PLUGIN) {
+        plugins.push(legacy({
```

Comment:
Could you add a dynamic import wrapped in a try / catch statement?
This would allow disabling the feature completely on internal clients by just uninstalling the optional dependency.

Reply:
done @alexisfontaine